### PR TITLE
Action buttons are displayed only for the active stage

### PR DESCRIPTION
**Pull request by lgalis**
Check the active stage for action button display.
Rename action transcript file to match the component name. \
Update icon in stage transcript.

Only the active stage has the Approve/Deny/Comment buttons:

Screenshot from 2019-05-08 09-52-33

Update icon for 'approved' action:
Screenshot from 2019-05-08 09-54-50

### CODE REVIEW

File: src/smart-components/request/request-detail/action-transcript.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ import {
...
   TextVariants
 } from '@patternfly/react-core';
-import { CheckIcon, CommentIcon, EnvelopeIcon, OutlinedTimesCircleIcon, AngleDoubleRightIcon } from '@patternfly/react-icons';
+import { CheckCircleIcon, CommentIcon, EnvelopeIcon, OutlinedTimesCircleIcon, AngleDoubleRightIcon } from '@patternfly/react-icons';
 
 const operationDisplayName = { memo: 'Comment from', approve: 'Approved By',
   deny: 'Denied by', notify: 'Notified By', skip: 'Skipped by' };
 
-const operationIcon = { memo: <CommentIcon/>, approve: <CheckIcon/>,
+const operationIcon = { memo: <CommentIcon/>, approve: <CheckCircleIcon fill="#32CD32"/>,
```
Comment (Hyperkid123) [inline src/smart-components/request/request-detail/action-transcript.js]: Just a small comment here. I think it the green color should be `#92D400`. You can see some color guide lines for PF here: https://www.patternfly.org/v4/design-guidelines/styles/colors

File: src/smart-components/request/request-detail/request-stage-transcript.js [hunk 2]
```diff
@@ -4,9 +4,10 @@ import { Title } from '@patternfly/react-core';
 import StageList from './stage-list';
 
 const RequestStageTranscript = ({ request }) => {
+  console.log('Debug - request', request);
```
Comment (Hyperkid123) [inline src/smart-components/request/request-detail/request-stage-transcript.js]: console log

Comment (lgalis) [reply src/smart-components/request/request-detail/request-stage-transcript.js]: Removed - thank you for noticing again!

File: src/smart-components/request/request-detail/action-transcript.js (hunk 1 above)
Comment (lgalis) [reply src/smart-components/request/request-detail/action-transcript.js]: You are right - I replaced this with the pf4 success color, and added the red one for 'denied'

**Comment by lgalis** [comment]
@Hyperkid123 - feedback addressed - thank you!

**Pull request merged**